_harp/public/pages/css.md

```diff
@@ -1,70 +1,25 @@
-<!--
-
-Table of Contents
-
-* Introduction
-* LESS as a Framework
-	* _explanation_
-	* _also support SASS and vanilla CSS if necessary_
-* Basics
-	* Ruleset Guidelines
-		* Basic Syntax
-			* _opening brace on same line as selector_
-			* _one space before opening brace_
-			* _properties and values on the same line_
-			* _one space after colon that follows property name_
-			* _each property/value declaration on its own line_
-			* _each property/value declaration indented_
-			* _closing brace on its own line_
-		* Supplemental Syntax
-			* _in rulesets with multiple selectors, each selector on its own line_
-	* Classes and IDs
-		* _IDs reserved for JS selectors_
-		* ID Naming Conventions
-			* _'#js-...'_
-		* Class Naming Conventions
-			*
-* Organization
-	* Table of Contents
-	* Comments
-		* Sectioning Comments
-		* Descriptive Comments
-	* Meaningful Whitespace
-		* Between Rulesets
-		* Delimiting Sections
-		* Aligning Values of Related Declarations (Optional)
-* File Structure
-	* Overview
-		* _app_
-		* _normalize_
-		* _tools_
-		* _globals_
-		* _components_
-	* App
-		* _table of contents, comments, and imports only&mdash;no rulesets_
-	* Normalize
-		* _required_
-		* _imported first_
-
-In progress...
-
--->
-
-### Introduction
-
-Cuberis projects typically use Less for CSS pre-processing. Less extends the CSS language, adding features that allow variables, mixins, functions and many other techniques that make CSS more maintainable, themable and extendable. Documentation on Less is available at [lesscss.org](http://lesscss.org).
-
-Projects in vanilla CSS or other pre-processors such as Sass are also supported, if necessary.
-
-Documentation for SASS can be found at http://sass-lang.com/
-
-Cuberis' CSS standards are heavily influenced by (**but differ from**) [Harry Roberts' CSS guidelines](http://cssguidelin.es/).
-
-The most common framework in Cuberis projects is Bootstrap, however, the is history and experience using foundation.
-
-### Rulesets
-
-The following terminology when discussing CSS rulesets is used at Cuberis:
+## Introduction
+
+Our preferred method of writing CSS is using [SASS](https://sass-lang.com/), a CSS pre-processing language that allows you to use variables, nested rules, functions and more. This helps keep large stylesheets well-organized in smaller files as well as improving maintainability for the life of a website.
```

Comment:
Super nitpicky... but can we change all occurrences of SASS to **Sass**?

Reply:
👍